docs/nydus-fscache.md

```diff
@@ -58,6 +58,26 @@ $ sudo yum update kernel --enablerepo Plus
 $ sudo reboot
 ```
 
+## Enable fscache
```

Comment:
If we have the step, is the previous `Prepare the kernel` step still needed?

Reply:
Sure. Enabling fscache for nydus requires a Linux kernel with $\geq$ 5.19 version and `cachefilesd` daemon startup. The `Perpare the kernel` step ensures we have the necessary environment for starting `cachefilesd` daemon, including a correct version of Linux kernel and necessary dependancies. However, this step won't start `cachefilesd` daemon automatically for some reason that I don't know. So we'd better check out whether the `cachefilesd` daemon has been started or not and follow the `Enable fscache` step that I added.